Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -367,6 +345,8 @@ void UpdateEmptyViewVisibility(bool isEmpty)
 					// the Forms layout for its content
 					_emptyViewFormsElement.Layout(_emptyUIView.Frame.ToRectangle());
 				}
+
+				_currentBackgroundIsEmptyView = true;
```

Comment:
Moved it a few lines down to align with `_currentBackgroundIsEmptyView = false;`.